### PR TITLE
NEWS: add some news about xbps

### PR DESCRIPTION
**Pull request by Duncaen**

### CODE REVIEW

**Comment by Duncaen** [comment]
@xtraeme anything I should add?

File: _posts/2019-06-26-xbps-news.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+In the last week we pushed a lot of XBPS updates, **a lot**.
+
+The previous post about XBPS 0.54 was rather short and I would like to show a few more new things.
+If you are interested in all changes, read the [NEWS](https://github.com/void-linux/xbps/blob/master/NEWS) file in the void-linux/xbps repository.
+
+## Performance improvements
+
+Some users might have noticed that removing the `plasma-desktop` package took ages, because of some dependency cycles.
+The dependency resolution algorithm has been completely rewritten and is now able to resolve the dependencies of `kde-plasma` with
```
Comment (xtraeme) [inline _posts/2019-06-26-xbps-news.md]: hmm looks incomplete?

Comment (Duncaen) [reply _posts/2019-06-26-xbps-news.md]: oh yes I didn't know what to write and skipped it and then forgot about it.
"within seconds", "in no time".

Comment (xtraeme) [reply _posts/2019-06-26-xbps-news.md]: Looks ok to me, albeit some grammar issues.

**Comment by xtraeme** [comment]
so why isn't this merged after 6 months... really?

**Comment by xtraeme** [comment]
come on buddy, ffs

**Comment by Duncaen** [comment]
No one reviewed the grammar and I forgot about it, at this point I'm thinking of just adding this to a post for the next release.

File: _posts/2020-01-30-xbps-news.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+ * Fixed an issue with happy eyeballs, where when a socket
+   fails within the connection delay it aborts with the error
+   instead of continuing with the next address. [jnbr](https://github.com/jnbr)
+
+### xbps-0.55 (2019-06-21):
+
+ * [xbps-install(1)](https://man.voidlinux.org/xbps-install.1): Added `-D, --download-only` flag to allow
+   downloading packages to the cache without attempting to
+   install them. [Toyam Cox]
```
Comment (Vaelatern) [inline _posts/2020-01-30-xbps-news.md]: `[Vaelatern](https://github.com/Vaelatern)`

File: _posts/2020-01-30-xbps-news.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+   [Chocimier](https://github.com/Chocimier)
+
+ * [xbps-checkvers(1)](https://man.voidlinux.org/xbps-checkvers.1): fixed a compile-time warning with glibc.
+   [xtraeme](https://github.com/xtraeme)
+
+ * [xbps-alternatives(1)](https://man.voidlinux.org/xbps-alternatives.1): check if alternative is a
+   symlink before removing it. [xtraeme](https://github.com/xtraeme)
+
+ * Try to find a usable shell instead of using `/bin/sh`
+   when executing package scripts. [Gottox]
```
Comment (Vaelatern) [inline _posts/2020-01-30-xbps-news.md]: `(https://github.com/Gottox)`

**Pull request merged**